back-end/hub-api/src/main/java/io/apicurio/hub/api/rest/impl/TemplatesResource.java

```diff
@@ -0,0 +1,1655 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.hub.api.rest.impl;
+
+import io.apicurio.hub.api.rest.ITemplatesResource;
+import io.apicurio.hub.core.beans.ApiDesignType;
+import io.apicurio.hub.core.beans.ApiTemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author c.desc2@gmail.com
+ */
+public class TemplatesResource implements ITemplatesResource {
+
+    private static Logger logger = LoggerFactory.getLogger(TemplatesResource.class);
+
+    private static final List<ApiTemplate> allTemplates = List.of(
```

Comment:
Let's store each template as a resource instead of inlining them in the Java code.  We can load the resources in a `static` intializer.